action.yml

```
@@ -45,6 +45,9 @@ inputs:
       and `CheckHostIP=no` to the SSH command line. Use the input `ssh-known-hosts` to
       configure additional hosts.
     default: true
+  ssh-user:
+    description: >
+      The user to use when connecting to the remote SSH host. By default 'git' is used.

```

```suggestion
  ssh-user:
    description: >
      The user to use when connecting to the remote SSH host. By default 'git' is used.
    default: git
```

Should we include the default in the `action.yml`?